app/Models/Course.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Course extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $fillable = [
+        'name',
+        'price',
+        'start_date',
+        'finish_date',
+        'description',
+        'teacher_name',
+    ];
+
+    public function user() 
+    {
+        return $this -> belongsToMany(User::class);
+    }
+
+    public function review() 
```

Comment:
`$this -> hasMany(Review::class);` trả ra 1 review hay nhiều reviews?

Reply:
nếu nhiều thì tên chưa đúng nhé, xem và sửa các phần tương tự nhé

Reply:
trả ra nhiều reviews ạ, em đã sửa lại rồi